baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/DeprecatedApiUsageTest.java

```diff
@@ -0,0 +1,254 @@
+/*
+ * (c) Copyright 2025 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.errorprone;
+
+import com.google.errorprone.CompilationTestHelper;
+import org.junit.jupiter.api.Test;
+
+public class DeprecatedApiUsageTest {
+
+    @Test
+    public void testNotDeprecatedMethodCall() {
```

Comment:
Let's maintain our style of not prefixing with `test` and writing test names as sanke_case_english_sentences